### PR TITLE
DOC-392: Reviewing system vars page

### PR DESCRIPTION
**Pull request by azhebel**

### CODE REVIEW

File: doc/source/wsrep-system-index.rst [hunk 1]
```diff
@@ -294,265 +436,454 @@ This variable is used to set the `notification command <http://galeracluster.com
...
+  Privileges and perimssions for running |Percona XtraBackup|
+  can be found `in Percona XtraBackup documentation
+  <https://www.percona.com/doc/percona-xtrabackup/2.4/using_xtrabackup/privileges.html>`_.
 
-  .. note:: This method is currently recommended if you have ``innodb-log-group_home-dir/innodb-data-home-dir`` in your config. Refer to :option:`sst-special-dirs` for more information.
+  This is the **recommended** and default method for |PXC|.
+  For more information, see :ref:`xtrabackup_sst`.
 
-* ``rsync``: Uses ``rsync`` to perform the SST, this method doesn't use the :variable:`wsrep_sst_auth`
+  .. note:: If you have ``innodb_data_home_dir`` and ``innodb_log_home_dir``
```
Comment (kennt-percona) [inline doc/source/wsrep-system-index.rst:817]: The sst-special-dirs option was removed (for 5.6 and 5.7), so this section can be removed.

File: doc/source/wsrep-system-index.rst [hunk 2]
```diff
@@ -294,265 +436,454 @@ This variable is used to set the `notification command <http://galeracluster.com
...
+   :default: ``0``
 
-This variable is used to control causality checks on certain statements.
+Controls cluster-wide causality checks on certain statements.
 Checks ensure that the statement is executed on a node
 that is fully synced with the cluster.
 
+.. note:: Causality checks of any type can result in increased latency.
+
 The type of statements to undergo checks
```
Comment (kennt-percona) [inline doc/source/wsrep-system-index.rst:887]: For the following section, do we want to include the values for 5,6, and 7?

Comment (azhebel) [reply doc/source/wsrep-system-index.rst:887]: @kennt-percona am I right that
5 = 1 AND 4
6 = 2 AND 4
7 = 1 AND 2 AND 4

Comment (kennt-percona) [reply doc/source/wsrep-system-index.rst:887]: @azhebel  Yes, that is correct, it's a combination of the bits

**Pull request merged**